app/Http/Controllers/Api/ExperimentController.php

```diff
@@ -188,22 +189,44 @@ public function saveExperimentResult(Request $request)
         $experimentResult = new ExperimentResult();
 
         $experimentId = $request->get('experiment_id');
+        $timeStarted = $request->get('time_started');
```

Comment:
You missed course_id  in the addition, please add in.

Reply:
To avoid redundancy, there is experiment_id and experiment belong to a course, I can trace back to get course_id that way.

Reply:
@bynalab I think he has a need for Course ID on the Frontend but let's hear from the horse's mouth. @saidabdul80 thoughts?

Reply:
Yes sir, so he can return course_id since there's a relationship between both